clients/gateway_test.go

```diff
@@ -26,14 +26,18 @@ func TestStateUpdateUnmarshal(t *testing.T) {
         }
       ]
     },
-    "nonces": {},
+    "nonces": { 
```

Comment:
The `jsonData` used here and in `TestGetStateUpdate` are the same thing so I think this update should be made there as well. 

I understand that we don't need `nonces` and `declared_contracts` for the test. But I think the data should still be updated to tally with the data source.

Reply:
Alternatively, we can have a global variable for the state update's `jsonData` so we don't have duplicate code.